CHANGELOG.md

```diff
@@ -206,6 +206,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Improve coverage of `self.log`-ing in any `LightningModule` or `Callback` hook ([#8498](https://github.com/PyTorchLightning/pytorch-lightning/pull/8498))
 
 
+- `self.log`-ing without a `Trainer` reference now raises a warning insttead of an exception ([#9733](https://github.com/PyTorchLightning/pytorch-lightning/pull/9733))
```

Comment:
```suggestion
- `self.log`-ing without a `Trainer` reference now raises a warning instead of an exception ([#9733](https://github.com/PyTorchLightning/pytorch-lightning/pull/9733))
```